pkgs/development/python-modules/shiboken6/default.nix

```diff
@@ -31,15 +31,18 @@ stdenv'.mkDerivation (finalAttrs: {
     (python.pythonOnBuildForHost.withPackages (ps: [ ps.setuptools ]))
   ] ++ lib.optionals stdenv.hostPlatform.isLinux [ autoPatchelfHook ];
 
-  buildInputs = [
-    llvmPackages.llvm
-    llvmPackages.libclang
-    python.pkgs.qt6.qtbase
-    python.pkgs.qt6.darwinVersionInputs
-    python.pkgs.ninja
-    python.pkgs.packaging
-    python.pkgs.setuptools
-  ];
+  buildInputs =
+    [
+      llvmPackages.llvm
+      llvmPackages.libclang
+      python.pkgs.qt6.qtbase
+      python.pkgs.ninja
+      python.pkgs.packaging
+      python.pkgs.setuptools
+    ]
+    ++ lib.optional stdenv.hostPlatform.isDarwin [
```

Comment:
Shouldn't be a list, right?

Reply:
No longer confident the fix is correct 🫠

Reply:
it's so annoying that `toString` doesn't care about nested lists. We should at some point make structured attrs default